common/socketoptions.hpp

```diff
@@ -166,7 +166,12 @@ SocketOption srt_options [] {
     { "tsbpddelay", 0, SRTO_TSBPDDELAY, SocketOption::INT, SocketOption::PRE },
     { "tlpktdrop", 0, SRTO_TLPKTDROP, SocketOption::BOOL, SocketOption::PRE },
     { "nakreport", 0, SRTO_NAKREPORT, SocketOption::BOOL, SocketOption::PRE },
-    { "conntimeo", 0, SRTO_CONNTIMEO, SocketOption::INT, SocketOption::PRE }
+    { "conntimeo", 0, SRTO_CONNTIMEO, SocketOption::INT, SocketOption::PRE },
+    { "lossmaxttl", 0, SRTO_LOSSMAXTTL, SocketOption::INT, SocketOption::PRE },
```

Comment:
I saw no desctiption of this in the pull request comment

Reply:
It's a minor change, this option was there since a long time (at least since the github version) and I forgot to expose it through socketoptions.